create files and path structure

MohammedHasan-99 opened the pull request:
relates #1

MohammedHasan-99 committed:
create files and path structure
relates #1

sara219 reviewed (changes requested):
- Change the name of `welcome_screen`  to `index.html`, also it's the only file that should be in the root! 

- Create a directory called HTML and move the reset of HTML files to it!

MohammedHasan-99 committed:
create file structure
rename welcome_screen to  index.html
rename welcome_screen.css yo style.css
create html directory and move the rest of html file to it
relates #1

MohammedHasan-99 committed:
I renamed home_screen.html to index.html and I moved the rest to html folder relates #1
